index.html

```diff
@@ -0,0 +1,229 @@
+            </li>
+            <li class="list-item-ii">
+                <p class="program-title fw-bold fs-6">CC Ignite</p>
+            </li>
+            <li class="list-item-iii">
+                <p class="program-description">Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+            </li>
+        </ul>
+        <button type="button" class="btn mt-5 mb-4">Learn Web Development</button>
+        <!-- for desktop -->
+        <div class="dt-courses">
+            <div class="dt-course-detail d-flex flex-column align-items-center">
+                <img class="program-image" src="images/lecturer.png" alt="Lecturer-Icon">
+                <p class="program-title fw-bold fs-6 py-1">Lecture</p>
+                <p class="program-description">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="dt-course-detail d-flex flex-column align-items-center">
+                <img class="program-image" src="images/picture.png" alt="Exhibition-Icon">
+                <p class="program-title fw-bold fs-6 py-1">CC Exhibition</p>
+                <p class="program-description">Appreciate various creations applying  CC license of artists, organized from Art Center Nabi.</p>
+            </div>
+            <div class="dt-course-detail d-flex flex-column align-items-center">
+                <img class="program-image" src="images/chat.png" alt="Forum-Icon">
+                <p class="program-title fw-bold fs-6 py-1">Forum</p>
+                <p class="program-description">Have the time to share your thoughts and opinions with experts for each topic.</p>
+            </div>
+            <div class="dt-course-detail d-flex flex-column align-items-center">
+                <img class="program-image" src="images/presentation.png" alt="Workshop-Icon">
+                <p class="program-title fw-bold fs-6 py-1">Workshop</p>
+                <p class="program-description">Try creating your own work using open source license rather than just watching.</p>
+            </div>
+            <div class="dt-course-detail d-flex flex-column align-items-center">
+                <img class="program-image" src="images/connection.png" alt="Network-Icon">
+                <p class="program-title fw-bold fs-6 py-1">CC Ignite</p>
+                <p class="program-description">Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+            </div>
+        </div>
+        <a href="#" class="courses-btn">See All Courses</a>
+    </section>
+    <!-- Featured Speakers -->
+    <section id="speakers" class="d-flex flex-column align-items-center px-2 py-4">
+        
+        <h3 class="fw-bold mb-5">Featured Instructors</h3>
+
+        <div class="instructors-list">
+            <!-- check javascript for dynamic data -->
+        </div>
```

Comment:
- [ ] As the last code reviewer said please follow the original design. The `desc2` description is bolder than the original design, please 🙏🏽 correct the font style to be similar to the original design.